cloud_chatops/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3
+WORKDIR /opt/cloud_chatops
```

Comment:
Under the docs:
https://hub.docker.com/_/python

They point people to `/usr/src/app` so this looks suspect? Any reason we've broken convention in this image

Reply:
No reason, I did not know (look) that it was convention.